### ISSUE TITLE
Any chance of an easier mode option?

### ISSUE THREAD

**Issue by 133794m3r** [00-Evan/shattered-pixel-dungeon#118]
I apparently have god awful luck with this game and the original if I'm lucky I get to floor 3 and then I die, the best i've ever done I think is floor 4 but both this game and the original I never get too far. I was wondering if there would ever be an "easier" mode option have it disable badges or whatever but the game atm is way too rng heavy and I don't understand how others can have such great runs without the rng screwing them over.

Anyway easier mode I was hoping for was something like making starting items slightly better like an extra damage for weapons/extra at the top end for the armors and 5 more health. Which should let people such as myself with my horrible luck they'd stand a chance of getting deeper into the game.

tl;dr easier mode option. By increasing stats of starter items by 1 level or so and starting health up by 5 should make it so that bad rng doesn't take you down as much and shouldn't be much coding. Disable badges during it whatever.

P.S. I compared this noe to the original one and the extra character things seem to give me a bit better options to keep going with the warrior's seal thing as that is supposedly the EZ mode class(which I still don't believe)

**Comment by 00-Evan** [comment]
I'm all for making the game more accessible, but not in a way that splits up the core experience. I would much rather work on improving tutorial functionality and tuning the overall difficulty curve.

RNG can definitely have a factor, but skill makes a big difference too.
